### DISCUSSION TITLE
Custom Links on Side Menu

### DISCUSSION THREAD

**Discussion by marcogandra** [farridav/django-jazzmin#277] [Q&A] [answered]
Please, I'm trying to add a Side Menu item, but it's not showing, my code is inside JAZZMIN_SETTINGS:
Could someone show me where my mistake? Thank you very much!

    #Custom links
    "custom_links": {
        "Other": [{
            # Any Name you like
            "name": "Site",
        
            # url name e.g. `admin: index`, relative urls e.g.` / admin / index` or absolute urls e.g. `https: // domain.com / admin / index`
            "url": "https://www.transcenderstudios.com",
        
            # any font-awesome icon, see list here https://fontawesome.com/icons?d=gallery&m=free&v=5.0.0,5.0.1,5.0.10,5.0.11,5.0.12,5.0.13, 5.0.2,5.0.3,5.0.4,5.0.5,5.0.6,5.0.7,5.0.8,5.0.9,5.1.0,5.1.1,5.2.0,5.3.0,5.3. 1,5.4.0,5.4.1,5.4.2,5.13.0,5.12.0,5.11.2,5.11.1,5.10.0,5.9.0,5.8.2,5.8.1,5.7.2, 5.7.1,5.7.0,5.6.3,5.5.0,5.4.2 (optional)
            "icon": "fas fa-comments",
        
            # a list of permissions the user must have to see this link (optional)
            # "permissions": ["books.view_book"]
    }]

**Comment by dbartenstein** [comment, marked as answer]
> Please, I'm trying to add a Side Menu item, but it's not showing, my code is inside JAZZMIN_SETTINGS:
> Could someone show me where my mistake? Thank you very much!
> 
> ```
> #Custom links
> "custom_links": {
>     "Other": [{
>         # Any Name you like
>         "name": "Site",
>     
>         # url name e.g. `admin: index`, relative urls e.g.` / admin / index` or absolute urls e.g. `https: // domain.com / admin / index`
>         "url": "https://www.transcenderstudios.com",
>     
>         # any font-awesome icon, see list here https://fontawesome.com/icons?d=gallery&m=free&v=5.0.0,5.0.1,5.0.10,5.0.11,5.0.12,5.0.13, 5.0.2,5.0.3,5.0.4,5.0.5,5.0.6,5.0.7,5.0.8,5.0.9,5.1.0,5.1.1,5.2.0,5.3.0,5.3. 1,5.4.0,5.4.1,5.4.2,5.13.0,5.12.0,5.11.2,5.11.1,5.10.0,5.9.0,5.8.2,5.8.1,5.7.2, 5.7.1,5.7.0,5.6.3,5.5.0,5.4.2 (optional)
>         "icon": "fas fa-comments",
>     
>         # a list of permissions the user must have to see this link (optional)
>         # "permissions": ["books.view_book"]
> }]
> ```

Hi @marcogandra: are you sure your app name is `Other`? I just gave it a try with one of my apps called `main` and it worked out of the box.
